### ISSUE TITLE
Release distributions3 0.2.1

### ISSUE THREAD

**Issue by alexpghayes** [alexpghayes/distributions3#92]
Prepare for release:

* [x] `git pull`
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_distributions3.html)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)
* [x] `devtools::build_readme()`
* [x] `urlchecker::url_check()`
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`
* [x] `revdepcheck::revdep_check(num_workers = 4)`
* [x] Update `cran-comments.md`
* [x] `git push`

Submit to CRAN:

* [x] `usethis::use_version('patch')`
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [x] `git push`
* [x] `usethis::use_github_release()`
* [x] `usethis::use_dev_version()`
* [x] `git push`

**Comment by alexpghayes** [comment]
cc @zeileis. This may either happen this week or may have to wait until after some travel. Feel free to ping me after Sept 20 if I've stalled!

**Comment by zeileis** [comment]
Wow, already submitted and accepted, that was fast!! :partying_face:

And also the GitHub pages (<https://alexpghayes.github.io/distributions3/>) are up to date. Wonderful, very much appreciated!

One more small idea (not urgent at all): The "Reference" overview on the GitHub page is currently a bit hard to follow, I think. I'll propose a grouping by type of functionality in a new PR. But as that is orthogonal to the CRAN release, it can be incorporated at any later point as well.

Thanks!

**Comment by zeileis** [comment]
Closing this issue because version 0.2.1 was successfully released on CRAN and development version on GitHub bumped to 0.2.1.9000.
